Misc/NEWS

```diff
@@ -10,6 +10,10 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-30876: Relative import from unloaded package now reimports the package
```

Comment:
What exactly command do you use? I used `python3 -m pip install --user blurb` and it installed blurb-1.0.